docs/api/cloudrootfs.md

```diff
@@ -0,0 +1,81 @@
+# cloud rootfs
+
+cloud rootfs package all the dependencies when run a kubernetes cluster needs.
```

Comment:
cloud rootfs "will/would" package all the dependencies refers to the kubernetes cluster requirements